DSharpPlus.Commands/CommandsExtension.cs

```diff
@@ -474,16 +469,6 @@ public override void Dispose()
         return;
     }
 
-    /// <summary>
-    /// The event handler used to log all unhandled exceptions, usually from when <see cref="commandErrored"/> itself errors.
-    /// </summary>
-    /// <param name="asyncEvent">The event that errored.</param>
-    /// <param name="error">The error that occurred.</param>
-    /// <param name="handler">The handler/method that errored.</param>
-    /// <param name="sender">The extension.</param>
-    /// <param name="eventArgs">The event arguments passed to <paramref name="handler"/>.</param>
-    private static void EverythingWentWrongErrorHandler<TArgs>(AsyncEvent<CommandsExtension, TArgs> asyncEvent, Exception error, AsyncEventHandler<CommandsExtension, TArgs> handler, CommandsExtension sender, TArgs eventArgs) where TArgs : AsyncEventArgs => sender.logger.LogError(error, "Event handler '{Method}' for event {AsyncEvent} threw an unhandled exception.", handler.Method, asyncEvent.Name);

```

Comment:
This event wasnt triggered anywhere?

Reply:
The event handler is no longer used no. This was invoked whenever the user's event handlers threw, however Aki has since removed that functionality long ago.